tests/functional/aws-node-sdk/test/multipleBackend/put/put.js

```diff
@@ -25,16 +25,29 @@ let s3;
 const awsTimeout = 30000;
 const retryTimeout = 10000;
 
+function _retryWithTimeouts(fn, numberRetry) {
```

Comment:
What is the second argument for? Won't it be `0` every time?

Reply:
on line 41 I call the same fn increasing the numberRetry argument

Reply:
ouch, yes sorry.

Reply:
I just thought you could have done like you did here: https://github.com/scality/S3/pull/996/files#diff-759441a4bc174becdbb400c56b574584R104. But it's the same.

Reply:
yes, I let the retryNumber be optional in the other functions because I was a bit lazy to edit all existing function calls to add the retryNumber argument.
In this case I had to replace all the setTimeout calls and remove the awsTimeout argument in the file so it wasn't too much extra work to expect the argument instead of letting it be optional.